src/Tools/dotnet-user-jwts/src/Commands/CreateCommand.cs

```diff
@@ -0,0 +1,204 @@
+                CommandOptionType.SingleValue);
+
+            cmd.HelpOption("-h|--help");
+
+            cmd.OnExecute(() =>
+            {
+                var (options, isValid) = ValidateArguments(
+                    app.ProjectOption, schemeNameOption, nameOption, audienceOption, issuerOption, notBeforeOption, expiresOnOption, validForOption, rolesOption, scopesOption, claimsOption);
+
+                if (!isValid)
+                {
+                    return 1;
+                }
+
+                return Execute(app.ProjectOption.Value(), options);
+            });
+        });
+    }
+
+    private static (JwtCreatorOptions, bool) ValidateArguments(
+        CommandOption projectOption,
+        CommandOption schemeNameOption,
+        CommandOption nameOption,
+        CommandOption audienceOption,
+        CommandOption issuerOption,
+        CommandOption notBeforeOption,
+        CommandOption expiresOnOption,
+        CommandOption validForOption,
+        CommandOption rolesOption,
+        CommandOption scopesOption,
+        CommandOption claimsOption)
+    {
+        var isValid = true;
+        var project = DevJwtCliHelpers.GetProject(projectOption.Value());
+        var scheme = schemeNameOption.HasValue() ? schemeNameOption.Value() : "Bearer";
+        var name = nameOption.HasValue() ? nameOption.Value() : Environment.UserName;
+
+        var audience = audienceOption.HasValue() ? audienceOption.Value() : DevJwtCliHelpers.GetApplicationUrl(project);
+        if (audience is null)
+        {
+            Console.WriteLine("Could not determine the project's HTTPS URL. Please specify an audience for the JWT using the --audience option.");
+            isValid = false;
+        }
+        var issuer = issuerOption.HasValue() ? issuerOption.Value() : DevJwtsDefaults.Issuer;
+
+        var notBefore = DateTime.UtcNow;
+        if (notBeforeOption.HasValue())
+        {
+            if (!ParseDate(notBeforeOption.Value(), out notBefore))
+            {
+                Console.WriteLine(@"The date provided for --not-before could not be parsed. Ensure you use the format 'yyyy-MM-dd [[[[HH:mm]]:ss]]'.");
+                isValid = false;
+            }
+        }
+
+        var expiresOn = notBefore.AddMonths(6);
+        if (expiresOnOption.HasValue())
+        {
+            if (!ParseDate(expiresOnOption.Value(), out expiresOn))
+            {
+                Console.WriteLine(@"The date provided for -expires-on could not be parsed. Ensure you use the format 'yyyy-MM-dd [[[[HH:mm]]:ss]]'.");
+                isValid = false;
+            }
+        }
+
+        if (validForOption.HasValue())
+        {
+            if (!TimeSpan.TryParseExact(validForOption.Value(), _timeSpanFormats, CultureInfo.InvariantCulture, out var validForValue))
+            {
+                Console.WriteLine("The period provided for --valid-for could not be parsed. Ensure you use a format like '10d', '24h', etc.");
```

Comment:
`{#>23}h` actually isn't valid, don't know if we want to specifically guard around that, or just not include it in the example text. Also 60s, 60m, etc.